src/main.cpp

```diff
@@ -307,6 +307,8 @@ int main(int argc, char *argv[])
 
     int errorcode = a.exec();
 
+    Settings::getInstance().saveGlobal();
```

Comment:
I'm not against it, but could you tell why moved?
Also add explantaion in commit text